scripts/tools/tools-schema.json

```diff
@@ -0,0 +1,82 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "JSON Schema for AsyncAPI tool discovery file.", 
+	"type": "object",
+	"required": [
+		"title",
+		"description",
+		"maintainers",
+		"links",
+		"filters"
+	],
+	"properties": {
+		"title": {
+			"type": "string",
+            "description": "Human-readable name of the tool that will be visible to people in the list of tools",
+            "examples" : [ "AsyncAPI Generator", "Cupid" ]  
+		},
+		"description": {
+			"type": "string"
+		},
+		"maintainers": {
+			"type": "array",
+			"items":{
+				"type": "string"
+			}
```

Comment:
```suggestion
			"type": "array",
			"items":{
				"type": "string"
			},
			"minItems": 1
```

Reply:
I'm still in confusion of having a `maintainers` array in schema or not. Let me confirm first and then will make the changes accordingly.

Reply:
yeah, maybe let us not over complicate if we have doubts, especially that we also agreed that to show this in UI is kinda complex enough to drop it for initial release.

so we dropping it from json and .asyncapi-tool? @magicmatatjahu @akshatnema

Reply:
For now, we have don't have anything to work with `maintainers` but what comes in my mind is that we are not defining any version for `.asyncapi-tool` file so if we come up with initial release and then change the `.asyncapi-tool` file to add maintainers, will it not make a problem for everyone so as to pass the schema and validate the file? 

My suggestion is that we leave this in the file for right now and then later on will make a UI for maintainers in the Tools card. What's your view? @derberg @magicmatatjahu

Reply:
yeah, we had discussion about version, and we decided that version is not so critical, it is all about just adding new code that will look for `maintainers` and that is it, and having proper fallback